addons/hr_timesheet_attendance/report/hr_timesheet_attendance_report_view.xml

```diff
@@ -25,6 +25,20 @@
             </field>
         </record>
 
+        <record id="view_hr_timesheet_attendance_report_tree" model="ir.ui.view">
+            <field name="name">view.hr.timesheet.attendance.report.tree</field>
+            <field name="model">hr.timesheet.attendance.report</field>
+            <field name="arch" type="xml">
+                <tree string="Timesheet Attendance Report">
```

Comment:
Hello Sir, In that case, shouldn't we update its XML-Id to a followed naming convention like
 model_name_view_view_type_{inherit}_{suffix}
as most of the existing views doesn't follow this.

Reply:
Well actually this changes an xml id and therefore requires a migration script... changing the string does not really require any update, that's why it could be done (to check with Ngr anyway).

Reply:
For now, i will update the string to upper case as you suggested